keps/sig-node/3008-qos-class-resources/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: QoS-class resources
+kep-number: 3008
+authors:
+  - "@marquiz"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
+creation-date: 2021-10-07
+reviewers:
+  - "@rphillips"
+approvers:
+  - "@sig-node-leads"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.26"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.26"
```

Comment:
BTW, this info may not still be current.

Reply:
Updated milestones, now targeting alpha for v1.28